tests/foreman/api/test_setting.py

```diff
@@ -1,13 +1,13 @@
 # -*- encoding: utf-8 -*-
 """Test class for Setting Parameter values
 
-:Requirement: Setting
+:Requirement: Test Cases for component Settings
```

Comment:
```suggestion
:Requirement: Setting
```
Please keep as above ..